README.md

```diff
@@ -5,6 +5,7 @@ Guides for getting things done, programming well, and programming in style.
 
 * [Best Practices](/best-practices)
 * [Code Review](/code-review)
+* [How to](/howto)
```

Comment:
This would be the only page with an explanation. Is there a better name for this?